pkg/cmd/clientConfig_test.go

```diff
@@ -0,0 +1,113 @@
+package cmd
+
+import (
+	"reflect"
+	"testing"
+
+	"regexp"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	kFake "k8s.io/client-go/kubernetes/fake"
+)
+
+func TestNewClientConfigCmd(t *testing.T) {
+	getFakeK8Client := func() kubernetes.Interface {
+		return &kFake.Clientset{}
+	}
+	fakeK8Client := getFakeK8Client()
+	tests := []struct {
+		name     string
+		k8Client kubernetes.Interface
+		want     *ClientConfigCmd
+	}{
+		{
+			name:     "new client config command",
+			k8Client: fakeK8Client,
+			want: &ClientConfigCmd{
+				k8Client: fakeK8Client,
+			},
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
I am not entirely sure what this test is testing. I know it is only a small test but struggling to see the benefit as it seems like the compiler should ensure this is the case

Reply:
This test is testing that NewClientConfigCmd returns fake client :) But you are right thats constructor testing. Does not mean any sense